apps/etterlatte-saksbehandling-ui/client/src/components/nyoppgavebenk/Oppgavelista.tsx

```diff
@@ -95,104 +49,15 @@ export const Oppgavelista = (props: { oppgaver: ReadonlyArray<OppgaveDTOny>; hen
     if (paginerteOppgaver.length === 0 && filtrerteOppgaver.length > 0) setPage(1)
   }, [paginerteOppgaver, filtrerteOppgaver])
 
+  useEffect(() => {
+    if (oppgaveErEndet) {
+      hentOppgaver()
+      setOppgaveErEndret(false)
+    }
+  }, [oppgaveErEndet])
+
```

Comment:
Hvorfor ikke bare kalle `hentOppgaver()`direkte i stedet for så mye frem og tilbake med `setOppgaveErEndret()`?

Reply:
Veldig godt spørsmål! Har fjernet det nå i [9bf7f80](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1922/commits/9bf7f80d70691615b0eddcbf5e8a8cda55333909)